<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleSettings</class>
 <widget class="QWidget" name="StyleSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Style settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>160</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>160</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Window</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Menu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Path toolbar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Side toolbar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>File view</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bookmark list</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Tab list</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Statusbar</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Others</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stack">
       <widget class="QWidget" name="pageWindow">
        <widget class="QLabel" name="label_29">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>57</width>
           <height>14</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Window</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageMenu">
        <widget class="QLabel" name="label_30">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>57</width>
           <height>14</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Menu</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pagePathToolbar">
        <widget class="QLabel" name="label_31">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Path toolbar</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageSideToolbar">
        <widget class="QLabel" name="label_34">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>111</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Side toolbar</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageFileView">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_13">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Color set on file view</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Hidden</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>68</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Cut list</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="edFvCuts"/>
         </item>
         <item row="8" column="0">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>4</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edFvExec"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Symbolic link</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="edFvHidden"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edFvSymbol"/>
         </item>
         <item row="14" column="0" colspan="2">
          <widget class="QCheckBox" name="chkFvStrikeOut">
           <property name="text">
            <string>Strike out cut list</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Executable</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Folder</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>File</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edFvFolder"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edFvFile"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Row spacing</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QLabel" name="label_19">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Misc.</string>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QLineEdit" name="edFvSpacing"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageBmkList">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_21">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Separator</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Foreground</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Background</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer_6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>4</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label_24">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Bookmark item</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Color</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Spacing</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1" colspan="2">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>178</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QLineEdit" name="edBmkColor">
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QLineEdit" name="edBmkSpacing"/>
         </item>
         <item row="1" column="1" colspan="2">
          <widget class="QLineEdit" name="edBmkSepFg">
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QLineEdit" name="edBmkSepBg">
           <property name="font">
            <font>
             <family>Monospace</family>
            </font>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageTabList">
        <widget class="QLabel" name="label_32">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>57</width>
           <height>14</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tab list</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="pageStatusbar">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="10" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>User</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Background</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QLineEdit" name="edStatGrpColor"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edStatFontName"/>
         </item>
         <item row="11" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Group</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLineEdit" name="edStatSizeColor"/>
         </item>
         <item row="13" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Date</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLineEdit" name="edStatUsrColor"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edStatFgColor"/>
         </item>
         <item row="13" column="1">
          <widget class="QLineEdit" name="edStatDateColor"/>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edStatFontSize"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Foreground</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="15" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Filename</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="16" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>10</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="14" column="1">
          <widget class="QLineEdit" name="edStatTimeColor"/>
         </item>
         <item row="9" column="1">
          <widget class="QLineEdit" name="edStatPmsColor"/>
         </item>
         <item row="14" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Time</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>4</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Statusbar</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Font size</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edStatBgColor"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Font name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="15" column="1">
          <widget class="QLineEdit" name="edStatFileColor"/>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Permission</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Color set on file information</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QCheckBox" name="chkStatUseWinBg">
           <property name="text">
            <string>Use window background</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageOthers">
        <widget class="QLabel" name="label_33">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>57</width>
           <height>14</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Others</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnTest">
       <property name="text">
        <string>Test</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
